models/cuboid/src/main.rs

```diff
@@ -1,6 +1,31 @@
-fn main() -> fj::Result {
-    let mut fj = fj::Instance::new();
-    let model = cuboid::model([3., 2., 1.], &mut fj.core);
-    fj.process_model(&model)?;
+use clap::Parser;
+use fj::{Args, Instance};
```

Comment:
The same comment from above, about making unrelated changes making things harder to review, applies here too.

In addition, I think this change makes the code clearly less readable here. The name of the `fj` crate is purposely short, to allow using things directly with the prefix. Now, it's harder to see where these types comes from, which is especially relevant because `Args` and `Instance` by themselves are such generic names.